### PR TITLE
chore: update addresses to reflect AaveV3 L2 encoding removal and KyberElastic Addition

### PR DESCRIPTION
**Pull request by SavDont**
## Checklist:

-   [ ] Add tests to cover changes as needed.
-   [ ] Update documentation as needed.
-   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: packages/contract-addresses/CHANGELOG.json [hunk 1]
```diff
@@ -1,4 +1,14 @@
 [
+	{
+		"version": "8.1.1",
+		"changes": [
+			{
+				"note": "Upgrade Polygon, Avalanche, Arbitrum, and Optimism FillQuoteTransformers to remove Aave V3 L2 Encoding and support KyberElastic",
+				"pr": 678
+			}
+		],
+		"timestamp": 1677693479
```
Comment (kyu-c) [inline packages/contract-addresses/CHANGELOG.json:10]: del: `timestamp`
* timestamp may prevent the package from being published.

**Review by kyu-c — APPROVED** [review]

File: packages/contract-addresses/CHANGELOG.json [hunk 2]
```diff
@@ -1,4 +1,14 @@
 [
+	{
+		"version": "8.1.1",
```
Comment (kyu-c) [inline packages/contract-addresses/CHANGELOG.json:3]: nit: 8.2.0 ?

File: packages/contract-addresses/addresses.json [hunk 3]
```diff
@@ -225,7 +225,7 @@
             "wethTransformer": "0x02ce7af6520e2862f961f5d7eda746642865179c",
             "payTakerTransformer": "0xa6c3ca183a67fcb4299fb4199c12ca74874ca489",
             "affiliateFeeTransformer": "0x3102aea537ecb6f164550b094663c82a8c53a972",
-            "fillQuoteTransformer": "0xfead4c05220a8fddc74008ae43199badc3becaf9",
+            "fillQuoteTransformer": "0xd140adb61d4e3e3978d4f32ac6b92240ff6e3a6e",
```
Comment (kyu-c) [inline packages/contract-addresses/addresses.json:228]: this doesn't look correct.

**Review by kyu-c — CHANGES_REQUESTED** [review]

File: packages/contract-addresses/addresses.json (hunk 3 above)
Comment (SavDont) [reply packages/contract-addresses/addresses.json:228]: https://optimistic.etherscan.io/tx/0xa5ae6ec17c5cff0b4f7ecf6c0b107a7d401c3ec68170692a75d2fd6b8480ce7b#eventlog
Heres my transaction that deployed the FQT. I'm not entirely sure why but it doesn't appear to be correct in optimistic etherscan. However, I yarn linked contract-addresses against the API and I was able to fill through this FQT.

Comment (kyu-c) [reply packages/contract-addresses/addresses.json:228]: discussed on slack

**Review by kyu-c — APPROVED** [review]

**Pull request merged**